esi_leap/api/controllers/v1/contract.py

```diff
@@ -149,7 +143,7 @@ def _contract_get_all_authorize_filters(cdict, r_project_id,
             policy.authorize('esi_leap:contract:get', cdict, cdict)
 
             if owner:
-                if r_project_id != owner:
+                if cdict['project_id'] != owner:
```

Comment:
If I understand the code correctly - can't a user in project X specify project Y for their project_id and owner filter, and avoid the policy check below?

Reply:
ah, I got confused - nevermind, although in the future this refactoring should be in a different PR